akvo/iati/exports/elements/document_link.py

```diff
@@ -18,9 +19,11 @@ def document_link(project):
     """
     document_link_elements = []
 
+    BASE_URL = "http://rsr.akvo.org"
```

Comment:
Why is the URL hardcoded here but not in the other document_link at akvo/iati/exports/org_elements/document_link.py?

Reply:
The code in both the contexts is slightly different. One of them has the request being passed as a parameter, but not in the other. Ideally, we should change all the elements to get the request as a parameter. I'll send that change in a separate PR.